pkg/odo/cli/component/watch.go

```diff
@@ -33,7 +33,7 @@ import (
 // WatchRecommendedCommandName is the recommended watch command name
 const WatchRecommendedCommandName = "watch"
 
-var watchLongDesc = ktemplates.LongDesc(`Watch for changes, update component on change.`)
+var watchLongDesc = ktemplates.LongDesc(`Watch for changes, update component on change. Watch doesn't provide support for git component.`)
```

Comment:
Can we assert the newly added string to avoid future regression ?

Reply:
Done